app/src/main/java/com/sun_asterisk/foodies/screen/main/MainActivity.kt

```diff
@@ -2,12 +2,48 @@ package com.sun_asterisk.foodies.screen.main
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.viewpager.widget.ViewPager
 import com.sun_asterisk.foodies.R
+import com.sun_asterisk.foodies.screen.favorite.FavoriteFragment
+import com.sun_asterisk.foodies.screen.home.HomeFragment
+import com.sun_asterisk.foodies.screen.main.layout_adapter.ViewPagerAdapter
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        setupTabLayout()
+    }
+
+    private fun setupViewPager(viewPager: ViewPager) {
+        val adapter = ViewPagerAdapter(supportFragmentManager)
+        adapter.apply {
+            addFragmentContainer(HomeFragment(), resources.getString(R.string.home))
+            addFragmentContainer(FavoriteFragment(), resources.getString(R.string.favorite))
+        }
+        viewPager.adapter = adapter
+    }
+
+    private fun setupTabIcons() {
+        val icons = resources.obtainTypedArray(R.array.layout_icons)
+        tabLayout.getTabAt(MainPage.HOME_PAGE)
+            .let { it?.setIcon(icons.getDrawable(MainPage.HOME_PAGE)) }
+        tabLayout.getTabAt(MainPage.FAVORITE_PAGE)
+            .let { it?.setIcon(icons.getDrawable(MainPage.FAVORITE_PAGE)) }
```

Comment:
Chưa tìm hiểu kĩ elvis operator trong kotlin nhé  
```suggestion
        val icons = resources.obtainTypedArray(R.array.layout_icons)
        tabLayout.getTabAt(MainPage.HOME_PAGE)?.setIcon(icons.getDrawable(MainPage.HOME_PAGE)
        tabLayout.getTabAt(MainPage.FAVORITE_PAGE)?.setIcon(icons.getDrawable(MainPage.FAVORITE_PAGE)
```